packages/e2e-tests/test-applications/standard-frontend-react/tests/fixtures/ReplayRecordingData.ts

```diff
@@ -0,0 +1,225 @@
+                      tagName: 'meta',
+                      attributes: { name: 'theme-color', content: '#000000' },
+                      childNodes: [],
+                      id: 7,
+                    },
+                    {
+                      type: 2,
+                      tagName: 'title',
+                      attributes: {},
+                      childNodes: [{ type: 3, textContent: '***** ***', id: 9 }],
+                      id: 8,
+                    },
+                  ],
+                  id: 4,
+                },
+                {
+                  type: 2,
+                  tagName: 'body',
+                  attributes: {},
+                  childNodes: [
+                    {
+                      type: 2,
+                      tagName: 'noscript',
+                      attributes: {},
+                      childNodes: [{ type: 3, textContent: '*** **** ** ****** ********** ** *** **** ****', id: 12 }],
+                      id: 11,
+                    },
+                    { type: 2, tagName: 'div', attributes: { id: 'root' }, childNodes: [], id: 13 },
+                  ],
+                  id: 10,
+                },
+              ],
+              id: 3,
+            },
+          ],
+          id: 1,
+        },
+        initialOffset: { left: 0, top: 0 },
+      },
+      timestamp: expect.any(Number),
+    },
+    {
+      type: 5,
+      timestamp: expect.any(Number),
+      data: {
+        tag: 'performanceSpan',
+        payload: {
+          op: 'memory',
+          description: 'memory',
+          startTimestamp: expect.any(Number),
+          endTimestamp: expect.any(Number),
+          data: {
+            memory: {
+              jsHeapSizeLimit: expect.any(Number),
+              totalJSHeapSize: expect.any(Number),
+              usedJSHeapSize: expect.any(Number),
+            },
+          },
+        },
+      },
+    },
+    {
+      type: 3,
+      data: {
+        source: 0,
+        texts: [],
+        attributes: [],
+        removes: [],
+        adds: [
+          {
+            parentId: 13,
+            nextId: null,
+            node: {
+              type: 2,
+              tagName: 'a',
+              attributes: { id: 'navigation', href: 'http://localhost:3000/user/5' },
+              childNodes: [],
+              id: 14,
+            },
+          },
+          { parentId: 14, nextId: null, node: { type: 3, textContent: '********', id: 15 } },
+          {
+            parentId: 13,
+            nextId: 14,
+            node: {
+              type: 2,
+              tagName: 'input',
+              attributes: { type: 'button', id: 'exception-button', value: '******* *********' },
+              childNodes: [],
+              id: 16,
+            },
+          },
+        ],
+      },
+      timestamp: expect.any(Number),
+    },
+    {
+      type: 3,
+      data: { source: 5, text: 'Capture Exception', isChecked: false, id: 16 },
```

Comment:
I'm not quite sure where this `text` comes from.. tried to replicate this in rrweb but couldn't. We should look into why this is not masked!

Reply:
(I would not block this PR on this, though)

Reply:
Hmm It doesn't look like a html element text, right? Maybe a console log? 

But yeah, I agree, it shouldn't block this PR

Reply:
Maybe it is related to what they pointed out here:
https://github.com/rrweb-io/rrweb/pull/1164#discussion_r1141271778

I will investigate this further!